policyengine_canada/parameters/gov/provinces/nt/tax/income/credits/age_credit/max_amount.yaml

```diff
@@ -0,0 +1,15 @@
+description: Northwest Territories provides this maximum amount under the age credit.
+values:
+  # Amount increases with inflation every year
+  2022-01-01: 7_635
+  2023-01-01: 8_116
+metadata:
+  unit: currency-CAD
+  label: Northwest Territories age credit maximum amount
+  reference:
+  - title: Income Tax Act - Age Credit Page 28, Section 2.2
```

Comment:
What number here corresponds to these values? Leave a comment indicating and add specificity to the section number/formula. You can also remove the page number from the title since it's in the URL.